packages/vue/src/components/molecules/SfAddToCart/SfAddToCart.vue

```diff
@@ -12,6 +13,7 @@
     </slot>
     <slot name="quantity-select-input" v-bind="{ qty }">
       <SfQuantitySelector
```

Comment:
On the desktop we rearranged elements using CSS so for the user who uses tabbing can be confusing. I will add an issue for that :)